VictorovaHomework2_1.cpp

```diff
@@ -0,0 +1,60 @@
+#include <iostream>
+
+int main() {
+  // 1
+  std::cout << "Please enter the number" << std::endl;
+  long long number{};
+  std::cin >> number;
+  int count{};
+  int sum{};
+  while (number != 0) {
+    sum += number % 10;
+    number /= 10;
+    count++;
+  }
+  std::cout << "The amount of digits=" << count << std::endl;
+
+  while (number > 0) {
+    sum += number % 10;
+
+    number /= 10;
+  }
+  std::cout << "Sum=" << sum << std::endl;
+  int average{};
+  average = sum / count;
```

Comment:
у вас будет целочисленное деление. 
Потом объединяйте объявление с инициализацией
```suggestion
 double average = sum / count;
```